src/app/concerns/concerns.interfaces.ts

```diff
@@ -1,6 +1,21 @@
 import { IGetRecordsResponse } from "../shared/records/records.interfaces";
 
-// TO DO: to be defined once BE is ready
+// TO DO: to be defined
 export interface IGetConcernsResponse extends IGetRecordsResponse {
   concernsInfo: any;
 }
+
+// TO DO: to be defined
+export interface IConcern {
+  doctorFirstName: string;
+  doctorLastName: string;
+  doctorStatus: string;
+  gmcReferenceNumber: string;
+  lastUpdatedDate: string;
+}
+
+// TO DO: to be defined
+export enum ConcernsFilterType {
+  UNDER_NOTICE = "underNotice",
```

Comment:
Just a *note: concerns are opened or closed. Concerns are raised anonymously if a doctor may be suspected of malpractice or abuse. database and service is different to. recommendation/revalidation. As previously suggested if initially separated the refactor can come about from combining common code as there are still unknowns. Speak with Ify she has it all mapped out in her head and could give insight as to filters, functionality etc. https://marvelapp.com/4dcc048/screen/68143896

Reply:
cheers good spot, completely forgot to do this. pr updated : )